docs/_providers/cloudflare.md

```diff
@@ -39,7 +39,9 @@ var REG_NAMECOM = NewRegistrar("name.com","NAMEDOTCOM");
 var CFLARE = NewDnsProvider("cloudflare.com","CLOUDFLAREAPI");
 
 D("example.tld", REG_NAMECOM, DnsProvider(CFLARE),
-    A("test","1.2.3.4")
+    A("test","1.2.3.4"),
+    A("www","1.2.3.4", {cloudlfare_proxy:"on"}),
+    ALIAS("@","test.example.tld",{cloudflare_proxy:"on"})
 );
 {%endhighlight%}
```

Comment:
I think a better practice is to define macros:

"""
var CF_PROXY_OFF = {'cloudflare_proxy': 'off'};     // Default/off.
var CF_PROXY_ON = {'cloudflare_proxy': 'on'};       // Proxy on.
var CF_PROXY_FULL = {'cloudflare_proxy': 'full'};   // Proxy on + railgun.
"""

Then you can do things like:

    A("test", "1.2.3.4", PROXY_ON),

(and switching things on and off become a lot easier).